storey/aggregations.py

```diff
@@ -584,21 +628,30 @@ def calculate_features(self, timestamp):
         if current_time_bucket_index < 0:
             return result
 
+        all_windows = []
+        if self.explicit_windows:
+            some_window = self.explicit_windows
```

Comment:
Maybe rename to something more meaningful other than "some_windows"

Reply:
but it is just a random window i need for the one call of round_up_time_to_window